### PR TITLE
[Update] Copy a Disk Over SSH

### PR DESCRIPTION
**Pull request by rsyracuse**
Clarification on copying/download language and process

### CODE REVIEW

**Comment by nmelehan** [comment]
Deploy preview for *nostalgic-ptolemy-b01ab8* ready!

Built with commit 28e68a758b13b9932df4901e0648642dc0531e3b

https://deploy-preview-3708--nostalgic-ptolemy-b01ab8.netlify.app

File: docs/guides/platform/disk-images/copying-a-disk-image-over-ssh/index.md [hunk 1]
```diff
@@ -26,11 +26,13 @@ If the amount of data on your disk is much less than the size of the disk, then
 
 ## Download a Disk over SSH
 
-### Boot into Rescue Mode
+This guide was written to support Downloading a disk over SSH from a  **Linode**, which will be used to Create and Send an image to a separate receiving system that you personally own. This means that the guide will have you switching between the execution of commands on both the **Origin Linode** as the origin of your disk image, and the **Receiving System** which will receive a copy of this image. You will see the terms Origin Linode and Receiving System used to describe these two separate environments.
```
Comment (leslitagordita) [inline docs/guides/platform/disk-images/copying-a-disk-image-over-ssh/index.md:29]: * Please remove usage of the future tense
* Downloading, Receiving System, and Origin Linode do not need to be capitalized.
* Rewrite suggestion for the first sentence: This guide shows you how to download a Linode's disk image over SSH to a separate receiving system, like a personal computer.

File: docs/guides/platform/disk-images/copying-a-disk-image-over-ssh/index.md [hunk 2]
```diff
@@ -26,11 +26,13 @@ If the amount of data on your disk is much less than the size of the disk, then
...
 
-1.  Prepare the receiving computer by verifying that SSH is installed. Most Linux/Unix-like systems include OpenSSH in their package base by default. If the receiving system is Microsoft Windows, there are multiple SSH solutions available such as [Cygwin and PuTTY](/docs/networking/ssh/using-ssh-on-windows).
+While this guide has been written to accommodate computers running Linux as their operating system, if the receiving system is Microsoft Windows, there are multiple SSH solutions available such as [Cygwin and PuTTY](/docs/networking/ssh/using-ssh-on-windows) which can alternatively be used to complete this process.
```
Comment (leslitagordita) [inline docs/guides/platform/disk-images/copying-a-disk-image-over-ssh/index.md:31]: missing the word running: "if the receiving system is running Microsoft Windows,"

File: docs/guides/platform/disk-images/copying-a-disk-image-over-ssh/index.md [hunk 3]
```diff
@@ -39,26 +41,24 @@ If the amount of data on your disk is much less than the size of the disk, then
 
 ### Copy and Download the Disk
 
-1.  Copy the disk over SSH from the Linode to the receiving machine. Replace `192.0.2.9` with the Linode's IP address and `/home/archive/linode.img` with the path where you want to store the disk.
+1.  Prepare the Receiving System that will be downloading the disk image by verifying that SSH is installed. Most Linux/Unix-like systems include OpenSSH in their package base by default.
```
Comment (leslitagordita) [inline docs/guides/platform/disk-images/copying-a-disk-image-over-ssh/index.md:44]: * Receiving System doesn't need to be capitalized 
* Please remove future tense.

File: docs/guides/platform/disk-images/copying-a-disk-image-over-ssh/index.md [hunk 4]
```diff
@@ -95,21 +95,24 @@ Once the copy has completed, verify it by mounting the image on the receiving ma
 
 ## Upload a Disk over SSH
 
-You may want to upload your disk image to a new server. For example, if you previously downloaded your Linode disk and deleted the Linode to halt billing on it, you can create a new Linode at a later date and upload the disk to resume your services.
+You may want to upload your disk image to a new server. For example, if you previously downloaded your Linode disk and deleted the Linode to halt billing on it, you can create a new Linode at a later date and upload the disk to resume your services. This section of the guide assumes that you will be creating a New Linode with the default Primary and Swap Disk as outlined in the Creating a Linode section of our [Getting Started](https://www.linode.com/docs/getting-started/#create-a-linode) guide.
```
Comment (leslitagordita) [inline docs/guides/platform/disk-images/copying-a-disk-image-over-ssh/index.md:98]: Remove future tense.

**Review by leslitagordita — CHANGES_REQUESTED** [review]

File: docs/guides/platform/disk-images/copying-a-disk-image-over-ssh/index.md [hunk 5]
```diff
@@ -95,7 +95,7 @@ Once the copy has completed, verify it by mounting the image on the receiving sy
 
 ## Upload a Disk over SSH
 
-You may want to upload your disk image to a new server. For example, if you previously downloaded your Linode disk and deleted the Linode to halt billing on it, you can create a new Linode at a later date and upload the disk to resume your services. This section of the guide assumes that you will be creating a New Linode with the default Primary and Swap Disk as outlined in the Creating a Linode section of our [Getting Started](https://www.linode.com/docs/getting-started/#create-a-linode) guide.
+In some cases, it is necessary to upload your disk image to a new server. For example, if you previously downloaded your Linode disk and deleted the Linode to halt billing on it, you can create a new Linode at a later date and upload the disk to resume your services. This section of the guide assumes that you are creating a New Linode with the default Primary and Swap Disk as outlined in the Creating a Linode section of our [Getting Started](https://www.linode.com/docs/getting-started/#create-a-linode) guide.
```
Comment (leslitagordita) [inline docs/guides/platform/disk-images/copying-a-disk-image-over-ssh/index.md:98]: New Linode --> new Linode
default Primary --> default primary
Swap Disk --> swap disk

**Review by leslitagordita — APPROVED** [review]

**Pull request merged**